### PR TITLE
Added TheAlgorithms resources to code and website URL to communities

### PR DESCRIPTION
**Pull request by msaf9**

### CODE REVIEW

File: courses/CSF211/README.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ This course is a prerequisite for:
 
 *   [Algorithms & data structures project, *William Fiset*](https://github.com/williamfiset/Algorithms)
 *   [WhatsAlgo, *Mohammed Shoaib*](https://github.com/Mohammed-Shoaib/WhatsAlgo)
+*   [TheAlgorithms, *Open Source resource for learning Data Structures & Algorithms*](https://github.com/TheAlgorithms "TheAlgorithms")
```
Comment (abxhr) [inline courses/CSF211/README.md:47]: Could you fix this link?
```suggestion
*   [TheAlgorithms, *Open Source resource for learning Data Structures & Algorithms*](https://github.com/TheAlgorithms)
```

File: courses/CSF211/README.md [hunk 2]
```diff
@@ -79,3 +80,4 @@ This course is a prerequisite for:
 *   [r/datastructures](https://www.reddit.com/r/datastructures/)
 *   [r/algorithms](https://www.reddit.com/r/algorithms/)
 *   [Data Structures and Algorithms, *CodeChef*](https://discuss.codechef.com/t/data-structures-and-algorithms/6599)
+*   [TheAlgorithms](https://the-algorithms.com/)
```
Comment (abxhr) [inline courses/CSF211/README.md:83]: This resource doesn't look like a discussion *community*. Could you replace it with another `Algorithms` community, where folks could ask questions and learn more about the domain?

**Review by abxhr — CHANGES_REQUESTED** [review]
Hello @msaf9! 👋
Thanks for your contribution! 💖
I've dropped two comments for the changes required. Let me know if you require any help

File: courses/CSF211/README.md (hunk 2 above)
Comment (msaf9) [reply courses/CSF211/README.md:83]: Yeah, my bad. I got it wrong, will provide the discord channel URL.

**Comment by msaf9** [comment]
I believe, this should be good to go. Please do let me know if there's something I need to modify.

**Review by abxhr — APPROVED** [review]
LGTM ✅

**Pull request merged**